packages/components/src/components/footer/footer.tsx

```diff
@@ -63,7 +63,7 @@ export class Footer {
   render() {
     return (
       <Host>
-        <div class="footer__container">
+        <div aria-label='a navigation footer' class="footer__container">
```

Comment:
I think the footer is an interactive element since it contains navigation links, but we could either add aria-label to the footer element only, or to the inside links. Generally it makes more sense to add aria label to the inside navigation links, but in our case, the links are slotted with different urls, so we can't "hard-code" a meaningful arial-label. So, we should keep the aria-label for the footer, and add aria-label to the social media links as they are not changeable, and additionally we should add a generic aria label to the nested links. I think this is probably the best we could do.

Reply:
the footer itself is not interactive though. we dont need any aria attributes when the element isnt interactive.

Reply:
There are navigation links inside the footer, and that makes it interactive, does it not? Usually the aria attributes on the links themselves should be enough, but in our case, I feel like they're not enough because the aria attribute of the Links cannot be specific enough, since they can be used with other components too, or stand alone. For that reason, I thought having the aria on the footer could add some more clarity. For me it's okay if you think they shouldn't be there?